app/api/activitylog/specs/activitylog.spec.js

```diff
@@ -47,53 +47,61 @@ describe('activitylog', () => {
       });
     });
 
-    it('should filter by method', async () => {
-      const { rows: entries } = await activitylog.get({ method: ['POST'] });
-      expect(entries.length).toBe(1);
-      expect(entries[0].method).toBe('POST');
-    });
+    describe('filtering', () => {
```

Comment:
Maybe switch the style to a more semantic approach, ie:
```
describe('when filtering')
  it('should return results by method')
```
So that it reads "when filtering it should return results by method".